### DISCUSSION TITLE
Home assistant / MQTT showing plato not SG

### DISCUSSION THREAD

**Discussion by liams1989** [universam1/iSpindel#625] [Q&A]
Hi, I'm wondering if I'm doing something wrong. 
In home assistant it's showing °P but the payload appears to look like an SG reading? 
![image](https://github.com/user-attachments/assets/15ce3292-9daf-48df-a68b-b0fc5347d88b)

**Comment by thegreatgunbantoad** [comment]
Double check the gravity calculation.

**Reply by thegreatgunbantoad** [reply]
Typically I use this for SG:
`(1.0000E-6*(tilt^3))+(-1.3137E-4*(tilt^2))+(6.9680E-3*tilt)+(9.0704E-1)`
I believe the Plato equivalent would be:
`(2.3351E-4*(tilt^3))+(-3.1541E-2*(tilt^2))+(1.7133E0*tilt)+(-2.3081E1)`

**Reply by liams1989** [reply]
I'll give both of the formulas a go and see what the MQTT output is. Thanks

**Reply by liams1989** [reply]
Nope. The MQTT is still:

Payload
unit_of_measurement: •P

And as far as I can see it cannot be changed when using MQTT
